chat-plugins/modlog.js

```diff
@@ -278,17 +302,30 @@ exports.commands = {
 		}
 
 		let addModlogLinks = Config.modloglink && (!hideIps || (targetRoom && targetRoom.isPrivate !== true));
-		// Let's check the number of lines to retrieve or if it's a word instead
 		let lines = 0;
-		if (!target.match(/[^0-9]/)) {
-			lines = parseInt(target || 20);
-			if (lines > RESULTS_MAX_LENGTH) lines = RESULTS_MAX_LENGTH;
+		let requestedLines = 0;
```

Comment:
I don't see a reason why `requestedLines` need to be in this scope. It would simplify your code to keep it in the `lines=` scope.